ruby-monday-blog/app/views/posts/_preview.html.erb

```diff
@@ -0,0 +1,22 @@
+<% hide_tags ||= false %>
```

Comment:
I'm not sure where this is coming from and why do we need it, could you elaborate please?